CheckDrive.Api/CheckDrive.Infrastructure/Sms/Factories/ISmsMetadataFactory.cs

```diff
@@ -0,0 +1,13 @@
+﻿using CheckDrive.Application.Models;
+using CheckDrive.Domain.Enums;
+
+namespace CheckDrive.Infrastructure.Sms.Factories;
+
+internal interface ISmsMetadataFactory
```

Comment:
how can  you use this interface with internal keyword in other layers. It was be public in my opinion